billingportal_configuration.go

```diff
@@ -99,12 +99,19 @@ type BillingPortalConfigurationFeaturesPaymentMethodUpdateParams struct {
 	Enabled *bool `form:"enabled"`
 }
 
+// Whether the cancellation reasons will be collected in the portal and which options are exposed to the customer
+type BillingPortalConfigurationFeaturesSubscriptionCancelCancellationReasonParams struct {
+	Enabled *bool     `form:"enabled"`
+	Options []*string `form:"options"`
```

Comment:
Shouldn't those be an enum?

Reply:
`stripe-go` doesn't have enums on params (yet)